packages/runtime/datastore/src/localChannelStorageService.ts

```diff
@@ -57,4 +63,26 @@ export class LocalChannelStorageService implements IChannelStorageService {
 
         return undefined;
     }
+
+    private readBlobSyncInternal(path: string, tree: ITree): IBlob | undefined {
```

Comment:
We can easily delete readSyncInternal() by changing readSync() to call into readBlobSyncInternal()